spec/factories/vacancies.rb

```diff
@@ -37,7 +44,7 @@
     expires_at { 6.months.from_now.change(hour: 9, minute: 0, second: 0) }
     hired_status { nil }
     include_additional_documents { false }
-    job_title { factory_sample(job_titles) }
+    sequence :job_title, job_titles.cycle
```

Comment:
Forces the factory to take iterate over the job titles list for each new vacancy instead of defaulting to the same first title on the list. This was causing system test failures due to ambiguous matching on Capybara when 2 vacancies shared the title.

Reply:
Ah nice one @scruti, thanks